src/content/en/updates/posts/2015/07/interact-with-ble-devices-on-the-web.markdown

```diff
@@ -44,7 +44,7 @@ finalized yet, the Chrome Team is actively looking for enthusiastic developers
 [feedback on the implementation](https://bugs.chromium.org/p/chromium/issues/entry?components=Blink%3EBluetooth).
 
 Web Bluetooth API is at the time of writing partially implemented in Chrome OS,
-Chrome for Android and Chrome for Linux behind an experimental flag. Go to
+Chrome for Android M, Linux and Mac behind an experimental flag. Go to
```

Comment:
I always forget this one. Thank You @jyasskin!
Fixed in https://github.com/google/WebFundamentals/pull/3147/commits/1fe13c7edc08cf9a94be9f25a6e88c2cf8764d98